compiler-rt/lib/orc/error.h

```diff
@@ -367,8 +367,8 @@ template <typename T> class ORC_RT_NODISCARD Expected {
   }
 
   union {
-    std::aligned_union_t<1, storage_type> TStorage;
-    std::aligned_union_t<1, error_type> ErrorStorage;
+    alignas(storage_type) char TStorage[sizeof(storage_type)];
+    alignas(error_type) char ErrorStorage[sizeof(error_type)];
```

Comment:
No change requested. FYI, pedantic reading of [[intro.object]/3](https://eel.is/c++draft/intro.object#3) suggests that `unsigned char` or `std::byte` should be used instead of `char` in these places.

Reply:
We have a bunch of existing llvm code that uses `char`, e.g.: https://github.com/llvm/llvm-project/blob/4244a91be268111dd133fa4863504c4840e5361c/llvm/include/llvm/Support/TrailingObjects.h#L355